IronFoundry.Container.Test/ContainerTests.cs

```diff
@@ -240,6 +273,69 @@ public void DisposesRunners()
             }
         }
 
+        public class GetInfo : ContainerTests
```

Comment:
I'd like to see some tests for the `Container.State` value.  We should have some on the Warden side.  There are specific states that we expect at different points and I want to make sure we codify that in tests.